GliaWidgets/Sources/ViewModel/EngagementViewModel.swift

```diff
@@ -222,11 +222,11 @@ class EngagementViewModel: CommonEngagementModel {
     }
 
     func updateScreenSharingState(to state: CoreSdkClient.VisitorScreenSharingState) {
-        screenShareHandler.updateState(to: state)
+        screenShareHandler.updateState(state)
```

Comment:
Why was the `to` removed? That way it was clear that the state was updated to `state`. Without it, you just read `updateState(state)` and it's not 100% clear what happens after updating state. Does `state` itself gets updated? Or the internal screenShareHandler state?

Reply:
I removed `to` due to compiler error, because now `updateState` is `var`, not method